bridges/FB2Bridge.php

```diff
@@ -194,7 +202,7 @@ private function buildContent($pageContent){
 		// /div>","replaceifexists
 		$regex = '/\\"html\\":(\".+\/div>"),"replace/';
 		preg_match($regex, $pageContent, $result);
-		return str_get_html(html_entity_decode(json_decode($result[1])));
+		return str_get_html(json_decode($result[1]));
```

Comment:
Thanks for the PR! :+1: 

I'm confused by this line though, why did you remove the `html_entity_decode`?

Reply:
When `html_entity_decode` is called there, it strips out a lot of information from the html, including the timestamp.

Moving the `html_entity_decode` calls to later in the process allowed me to extract the timestamps.

Reply:
Ah, I see now. Thanks for the clarification :+1: